src/game/enemy/systems.rs

```diff
@@ -201,4 +304,62 @@ pub fn catch_shot_event(
             }
         }
     }
-}
\ No newline at end of file
+}
+
+/// Enemy changes sprite sheet to shooting and triggers shot event
+pub fn enemy_shoot_player(
+    mut commands: Commands,
+    mut enemy_q: Query<(Entity, &Transform), (With<Enemy>, Without<Firing>)>,
+    enemy_handles: Res<EnemyHandles>,
+    mut enemy_shot_player_event_writer: EventWriter<EnemyShotPlayerEvent>,
+    keyboard_input: Res<Input<KeyCode>>, // Remove later
+) {
+    if keyboard_input.just_pressed(KeyCode::F2) {
```

Comment:
should probably at least leave a TODO here or even an issue to update this at some point, if we dont have a good way to automate this now

Reply:
I could implement something, but wasn't sure how to yet. Do we just want to have it on a set time or do certain conditions need to be met. Like within a certain distance, nothing on the map in the way, etc etc

Reply:
id think distance would be the chief factor (really mostly checking if they are on screen or not). Distance could then factor in to the chance of the shot actually succeeding

Reply:
I'll try making something, but we will probably have to get a level started and play it a bunch to tweak it to feel good